awx/ui_next/src/screens/InstanceGroup/shared/ContainerGroupForm.jsx

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import { func, shape } from 'prop-types';
+import { Formik, useField } from 'formik';
+import { withI18n } from '@lingui/react';
+import { t } from '@lingui/macro';
+import { Form, FormGroup } from '@patternfly/react-core';
+import { jsonToYaml } from '../../../util/yaml';
+
+import FormField, {
+  FormSubmitError,
+  CheckboxField,
+} from '../../../components/FormField';
+import FormActionGroup from '../../../components/FormActionGroup';
+import { required } from '../../../util/validators';
+import {
+  FormColumnLayout,
+  FormFullWidthLayout,
+  FormCheckboxLayout,
+  SubFormLayout,
+} from '../../../components/FormLayout';
+import CredentialLookup from '../../../components/Lookup/CredentialLookup';
+import { VariablesField } from '../../../components/CodeMirrorInput';
+
+function ContainerGroupFormFields({ i18n }) {
+  const [credentialField, credentialMeta, credentialHelpers] = useField(
+    'credential'
+  );
+  const [overrideField] = useField('override');
+
+  return (
+    <>
+      <FormField
+        name="name"
+        id="container-group-name"
+        label={i18n._(t`Name`)}
+        type="text"
+        validate={required(null, i18n)}
+        isRequired
+      />
+      <CredentialLookup
+        label={i18n._(t`Credential`)}
+        credentialTypeKind="kubernetes"
+        helperTextInvalid={credentialMeta.error}
+        isValid={!credentialMeta.touched || !credentialMeta.error}
+        onBlur={() => credentialHelpers.setTouched()}
+        onChange={value => {
+          credentialHelpers.setValue(value);
+        }}
+        value={credentialField.value}
+        required
+        tooltip={i18n._(
+          t`Credential to authenticate with Kubernetes or OpenShift.  Must be of type "Kubernetes/OpenShift API Bearer Token”.`
+        )}
+      />
+
+      <FormGroup
+        fieldId="container-groups-option-checkbox"
+        label={i18n._(t`Options`)}
+      >
+        <FormCheckboxLayout>
```

Comment:
@nixocio 

In terms of functionality, this is very similar to the notification config overrides we do on the notification form and the UX should be consistent for both features.

- In the 3.8 ui we use a toggle-switch for container groups and notifications
- The mockups [show a checkbox]( https://github.com/ansible/awx/pull/7967/files#diff-71e62927b1997ff95ae847242fe83b6cR59) for container groups for but don't specify any UX for notifications
- [We intend to use a switch](https://github.com/ansible/awx/pull/7967/files#) for notifications in ui_next


Personally, I prefer the switch but I think the important thing is that we're consistent with the type of input field we use for configuration overrides.

cc @trahman73 @mabashian @keithjgrant  thoughts?

Reply:
There was a slack discussion about the need to use `checkbox`. I think the mock up was updated upon the outcome of that discussion.

Reply:
:+1: Nice. Sounds good to me.